lib/schemaUtils.js

```diff
@@ -196,6 +196,13 @@ function safeSchemaFaker (oldSchema, resolveTo, resolveFor, parameterSourceOptio
     });
   }
 
+  if (schemaFormat === 'xml') {
```

Comment:
@aman-v-singh We shouldn't have this key specific to `XML` schema format. I'd suggest if we can use value of `schemaFormat` itself, it'd be good classification rather than special handling in XML case.

Also `schemaFormatXML` and  `schemaFormatDEFAULT` both contains `schemaFormat`, which is not needed and just adds more data. We can avoid using common keyword here.

Reply:
@VShingala I was following [this](https://github.com/postmanlabs/openapi-to-postman/blob/5929ebdd501aa23c557043d01ceb1d0763871990/lib/schemaUtils.js#L179-L190) pattern.
So should I change it for the `resolveTo` parameter also or keep it the same?

Reply:
@aman-v-singh Oh! Just noticed that possible value for `schemaFormat` can only be one of `default` or `xml`. I was under assumption that it can be more. In that case this seems fine.

Also, we can keep  `schemaFormatXML` and `schemaFormatDEFAULT`, it's working as good separators between each part of key.